server/service/instance.go

```diff
@@ -89,43 +89,47 @@ func (s *InstanceService) Register(ctx context.Context, in *pb.RegisterInstanceR
 	remoteIP := util.GetIPFromContext(ctx)
 
 	if err := Validate(in); err != nil {
-		util.Logger().Errorf(err, "register instance failed, invalid parameters, operator %s.", remoteIP)
+		util.Logger().Errorf(err, "register instance failed, invalid parameters, operator %s", remoteIP)
```

Comment:
Can we know about the invalid parameters details here?

Reply:
the err object will print the details